CRM/Volunteer/Hook.php

```diff
@@ -0,0 +1,13 @@
+<?php
+class CRM_Volunteer_Hook
+{
+  static $_nullObject = NULL;
+
+  static function runDefaultSettingsHook(&$defaults) {
```

Comment:
By convention this method take the name of the hook, minus the prefix. I think that means we'd call it _volunteer_projectDefaultSettings_ or _projectDefaultSettings_. I think I prefer the latter, since _volunteer_ is already in the class name.